stratosphere/pm/source/pm_main.cpp

```diff
@@ -36,7 +36,7 @@ extern "C" {
 
     u32 __nx_applet_type = AppletType_None;
 
-    #define INNER_HEAP_SIZE 0x30000
+    #define INNER_HEAP_SIZE 0x40000
```

Comment:
Interested in knowing what caused the heap size increase.

Reply:
Addition of the title tracking map. My math said that assuming 4000 unique switch games all launched without reboot, 0x10000 of memory should cover us.